README.md

```diff
@@ -1,2 +1,15 @@
-# My Lifelong Learning Lab
-Repo for code related to My Lifelong Learning Lab
+##What is My Lifelong Learning Lab?
+
+A part of the [FLOE Preference Exploration and Self-Assessment](https://wiki.fluidproject.org/display/fluid/%28Floe%29+Preference+Exploration+and+Self-Assessment) work.
+
+##Building
+
+Requirements:
+* `npm`
+
+Steps:
+* `npm install`
+    * postinstall should run the `grunt copy:frontEndDependencies` task after installation completes
```

Comment:
I suspect we probably shouldn't run `copy:frontEndDependencies` at all as a post-install, because there are scenarios where you wouldn't want this to happen (like if you were doing a clean install, as you say).

Since the aim here is to use Grunt tasks to avoid having to use a more complex means of building the project for development and demos, there should probably be a custom 'install' task that does something like:

- run npm
- copy the front end dependencies
- remove the node_modules directory